pkg/ccl/crosscluster/logical/create_logical_replication_stmt.go

```diff
@@ -195,6 +197,15 @@ func createLogicalReplicationStreamPlanHook(
 		}
 		streamAddress = crosscluster.StreamAddress(streamURL.String())
 
+		cleanedURI, err := cloud.SanitizeExternalStorageURI(from, nil)
+		if err != nil {
+			return err
+		}
+		cleanedURI, err = changefeedbase.RedactUserFromURI(cleanedURI)
```

Comment:
Do we need `RedactUserFromURI` ? We're redacting the password in `SanitizeExternalStorageURI` but user seems useful, no?